.circleci/config.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           name: "Create pre-commit-cache-key.txt"
           command: |
             cp .pre-commit-config.yaml /tmp/pre-commit-cache-key.txt
-            python --version --version | sed 's/^/# /' >> /tmp/pre-commit-cache-key.txt
+            python --version | sed 's/^/# /' >> /tmp/pre-commit-cache-key.txt
```

Comment:
@kaxil this needed two `--version` strings:
```
$ python --version
Python 3.8.5
$ python --version --version
Python 3.8.5 (default, Jan 27 2021, 15:41:15)
[GCC 9.3.0]
```

Reply:
oh wow ! Didn't know about it

Reply:
Go for it -- I didn't know. TIL:

```
-V     : print the Python version number and exit (also --version)
         when given twice, print more information about the build
```

Reply:
yeah, the syntax is a bit ambiguous. I feel like the extra info should be behind a more descriptive flag like `--build-info`

Reply:
https://github.com/astronomer/airflow-chart/pull/224